spec/struct.dd

```diff
@@ -176,7 +176,18 @@ $(H2 $(LNAME2 struct_layout, Struct Layout))
     the first field and the start of the object.
     )
 
-    $(P Structs with no fields of non-zero size (aka $(I Empty Structs)) have a size of one byte.)
+    $(P $(DDSUBLINK spec/attribute, linkage, `extern(D)`) structs with no fields of non-zero
```

Comment:
Avoid the double negation “no … with non-…”ł and instead use “all … with …”.
```suggestion
    $(P $(DDSUBLINK spec/attribute, linkage, `extern(D)`) structs with all fields of zero
```
Mathematically speaking, any _all_ statement about things is vacuously true if there are none. Most people who read this part of the spec will understand it, but if you’re in doubt, you could do this instead:
```suggestion
    $(P $(DDSUBLINK spec/attribute, linkage, `extern(D)`) structs with no fields or all fields of zero
```